keyboards/handwired/unk/rev1/rev1.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "unk.h"
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
This layout macro and the info.json doesn't seem to match the image you have in your repo. You can use `KC_NO` here to block out nonexistent matrix positions, so that you don't need to do it in the keymap.c.

Reply:
I need to leave these as empty to maintain future pluggable/optional `B` key on the right half. Is it okay to leave it as is?
 
I haven't update the repository yet. Here is the original layout, twww.keyboard-layout-editor.com/#/gists/457787e3b1938c4cbe9d65c7b7bd31ef. On keyboard-layout-editor.com I'm using empty and ghosted keys like this.

![Screen Shot 2020-06-29 at 21 47 43](https://user-images.githubusercontent.com/2534060/86020547-3e16fd00-ba52-11ea-80c1-a9c6da4a62d7.png)

This will resulted with 80 keys matrix as defined on `rev1.h` and `keymap.c`. Is it right?

Reply:
@fauxpark If I'm using 70 keys on info.json, the linting check will be failed since it check it against the matrix in the code. Can I use info.json with 80 keys and empty-ghosted keys layout like above?

Reply:
> future pluggable/optional B key on the right half

I'm not really sure what you mean here.

Reply:
> I'm not really sure what you mean here.

Some people used to type B letter with their right hand. I'm developing a single key module that can be attached to the right half of the keyboard. These empty keys are spared to handle this kind of need.

Reply:
Then you would add another layout macro with that key added. The whole point of them is to remove dummy keycodes like this.

Reply:
Oh okay. Let me try to resolve this. Thank you for your feedbacks!